algolia.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'rubocop', '<= 0.82.0'
 
-  spec.add_dependency 'faraday', '~> 0.15'
+  spec.add_dependency 'faraday', ['>= 0.15', '< 2.0']
```

Comment:
Does version `1.x` have backwards compatibility for version `0.x`? 🤔

Reply:
it doesn't break anything we're using!

Reply:
How did you test that? 🙂

Reply:
Okay, great! 😄  Thank you for the explanation 🙂